src/scss/custom/_tables.scss

```diff
@@ -10,3 +10,8 @@ td {
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+// this override is counter cross-browser inconsistencies with CSS Grid
```

Comment:
Is there a link somewhere outlining these inconsistencies? Like an SO link or something...

Might be nice to put in here for future devs.

Reply:
No, I didn't find any related articles.